algorithms/CPlusPlus/Arrays/segregate0and1.cpp

```diff
@@ -0,0 +1,48 @@
+/*
+Description: A program to segregate the array of 0s and 1s
+
+Approach: Counting the number of 0s present in the array and saving it in variable z. For the first z indexes, putting 0 and for the remaining indexes putting 1.
```

Comment:
It would be readable if you use multiple lines instead of one single line to address this comment,
so no one has to drag their mouse to the end or just move the scroll bar horizontally just to read it. 
```suggestion
Approach: Counting the number of 0s present in the array and saving 
it in variable z. For the first z indexes, putting 0 and for the remaining 
indexes putting 1.
```